azext_iot/_help.py

```diff
@@ -1133,6 +1140,13 @@
 ] = """
     type: command
     short-summary: Get the details of an enrollment group in an Azure IoT Hub Device Provisioning Service.
+    examples:
+    - name: Basic usage
+      text: >
+        az iot dps enrollment-group show --dps-name {dps_name} -g {resource_group} --enrollment-id {enrollment_id}
+    - name: Include attestation information in results
```

Comment:
Are there additional elements of attestation information in non symmetric key based attestation? Wondering if the example description should be scoped to symmetric key. If there are useful infos for the other types then carry on.